modules/ROOT/pages/api-groups-manage-api-group-instance-contracts.adoc

```diff
@@ -0,0 +1,80 @@
+
+When an application has multiple contracts with the same API instance, only one contract is active at any given time. This is known as _Applied contracts_.
+
+.The screen capture displays the contract levels and the contract status for the API instance contracts.
+image:api-groups-instance-contracts.png[API Group Instance Contracts,95%,100%]
+
+<1> The contracts for an API instance at the instance-level
+
+<2> The API Group for which the contracts are displayed
+
+<3> The applications that currently have a contract with the API instance
+
+<4> The contract that is currently active
+
+<5> The contracts for an API instance at the API Group-level
+
+An application can have multiple contracts with the different APIs in an API Group. You can view the the API instance contracts at the API Group-level. The contracts are sorted by the applications that are in contract with the API Group. The contract that is currently active within an API Group is displayed as `Applied`.
+
+When an application has multiple contracts with the same API, the least restrictive tier compared with other contract tiers of the same API is set to `Applied`. Only A contract can be set to Applied status only the contract is first approved. Because a revoked contract is no longer active, it cannot be set as `Applied`.
+
+== Applied Contracts Example
+
+To better understand how a contract is set to Applied if there are multiple contracts between an API and an application, let us look at the following example.
+
+Good Weather requests access to API `Facebook` for an application named `Contacts`. A contract is created between `Contacts` and `Facebook`. Now, the consumer uses the same application `Contacts` to request access to an API Group instance named `Social Chatter` which consists of the `Facebook` API among others. A contract is created between `Contacts` and `Social Chatter`.
+
+In this scenario, the application `Contacts` has two contracts:
+
+* One directly to API `Facebook`
+* One indirectly with API `Facebook` because of the contract with API Group instance `Social Chatter`
+
+Since `Contacts` has two contracts, now the question is which contract will be set to `Applied`. Let us say that the:
+
+* API contract tier: 100 requests per minute
+* API Group contract: 10 requests per minute
+
+In this scenario, the API contract will be set to `Applied` because it is the least restrictive tier. The user gets 90 requests more per minute than the API Group contract.
+
+Note that a contract is applied for an API, and not for an API Group instance. For example, if the consumer creates a contract between the API Group instance `Social Chatter` and another application `Interests`:
+
+* For `Contacts` and API `Facebook`, the applied contract will be the API contract.
+* For `Interests` and API `Facebook`, the applied contract will be the API Group instance contract because it is the only contract between the application and the API. 
```

Comment:
```suggestion
* For `Interests` and API `Facebook`, the applied contract is the API Group instance contract because it is the only contract between the application and the API. 
```

Reply:
Done! Thanks!